app/src/main/res/values-de/strings.xml

```diff
@@ -295,7 +296,25 @@
     <string name="settings_privacy">Hinweise zum Datenschutz</string>
     <string name="settings_terms_of_use">Nutzungsbedingungen</string>
 
-    <string name="settings_send_app_feedback">Feedback zur App senden</string>
+    <!-- -->
+    <string name="helpdesk_topic_technical_question">Technische Frage</string>
+    <string name="hint_issue">Anliegen</string>
+    <string name="helpdesk_info_text">Bitte beschreiben Sie Ihr Anliegen mit Hilfe des Formulars unten. Informationen über Ihr Mobilgerät und Ihre E-Mail-Adresse werden automatisch mit Ihrer Anfrage übermittelt.</string>
```

Comment:
First part only `Bitte beschreiben Sie Ihr Anliegen.`

Automatic email input is not always guaranteed. Text should adjust based on case.